hoomd/write/table.py

```diff
@@ -176,24 +176,30 @@ class _TableInternal(_InternalAction):
 
     def __init__(self,
                  logger,
-                 output=stdout,
+                 output=stdout or str,
```

Comment:
This is wrong. or is a Boolean operator and will make output either `True` or `False`.
```suggestion
                 output=stdout
```